tests/test_notify_slack.py

```diff
@@ -183,7 +183,7 @@ def should_notify_slack_when_step_function_failed(
 
         # Then assert there is 9 slack_sdk message 'blocks' sent to webhook url
```

Comment:
This comment no longer corresponds with the block count. Same for the other test.